s2i-java/Dockerfile

```diff
@@ -0,0 +1,98 @@
+FROM rhel:7.2
+
+ENV MAVEN_VERSION="3.3.3" \
+    JOLOKIA_VERSION="1.3.5" \
+    PATH=$PATH:"/usr/local/s2i" \
+    AB_JOLOKIA_CONFIG="/opt/jolokia/jolokia.properties" \
+    AB_JOLOKIA_AUTH_OPENSHIFT="true" \
+    JAVA_DATA_DIR=/deployments/data
+
+# Some version information
+LABEL io.k8s.description="Platform for building and running plain Java applications (fat-jar and flat classpath)" \
```

Comment:
I think we should have a cautionary message. but lets discuss how we want to implement this with the team